.github/workflows/ci.yml

```diff
@@ -79,13 +84,15 @@ jobs:
           # Workaround for https://github.com/actions/virtual-environments/issues/1811
           # Remove broken openssl and python packages, not used by Conscrypt builds
           # TODO(prb): remove when no longer required.
-          brew uninstall openssl@1.0.2t
-          brew uninstall python@2.7.17
-          brew untap local/openssl
-          brew untap local/python2
-          brew update
-          brew install ninja
-          echo "JAVA_HOME=$(/usr/libexec/java_home -v 1.8)" >> $GITHUB_ENV
```

Comment:
Isn't this still needed?  Or do the setup-java actions you added above take care of it?

Reply:
My understanding was it wasn't, but that was mainly because it looks like after they are applied the JAVA_HOME becomes sticky anyway

![image](https://user-images.githubusercontent.com/231923/102975201-960d5880-44f7-11eb-9112-462043837f07.png)